index.js

```diff
@@ -0,0 +1,292 @@
+const NOTES_DOM = {
+    notes_board:document.querySelector("#tasksBoard"),
+    note_name:document.querySelector("#noteName"),
+    note_text:document.querySelector("#noteText"),
+    note_date:document.querySelector("#dateToDo"),
+    note_time:document.querySelector("#timeToDo"),
+    add_button:document.querySelector("#addNoteButton"),
+    reset_button:document.querySelector("#resetFormButton")
+}
+
+NOTES_DOM.add_button.addEventListener("click",saveStickyNOTE)
+
+let arrayOfNotes = [];
+
+function getSelectedValue() {
+let sortByVal = document.querySelector("#sortBy").value
```

Comment:
1. you could have put this in your NOTES_DOM to have a constant reference
2. even better since you are in an onchange event handler then "this" points to the #sortBy element so you can change this code to:
let sortByVal =  this.value